addons/l10n_din5008/static/src/scss/report_din5008.scss

```
@@ -6,11 +6,11 @@
     &.header {
         table {
             width: 100%;
-            img, h3, td, tr {
+            img, div.h3, td, tr {
                 padding: 0;
                 margin: 0;
             }
-            h3 {
+            div.h3 {
```

```suggestion
            h3, div.h3 {
```
Same comment as above.